### ISSUE TITLE
Can maxwell bootstrap from a binlog file?

### ISSUE THREAD

**Issue by kevinclcn** [zendesk/maxwell#1070]
I use maxwell to do synchronization between two databases. Once the maxwell get down and I can only sync all the updates or new insert rows with the current bootstrap. I cannot remove the deleted rows in the target database. 

Will maxwell support the binlog file bootstrap? Or is there any other solutions? 

Thanks!

**Comment by osheroff** [comment]
I'm not really sure what you're asking.  maxwell isn't *exactly* designed for this use case.  If you're outputting to kafka or another durable stream, after the bootstrap is complete maxwell will play back any changes (including deletes) that occurred while it bootstrapped.  But again, I'm not sure what your question really is.

**Comment by kevinclcn** [comment]
@osheroff Sorry for the confusion. 

We use Alibaba RDS in our system and it rotates the binlog every several hours. We had experienced that maxwell stop to work but the ping API works properly. So when we realized that maxwell has stopped to work, the binlog has been rotated to Aliababa's OSS (an Amazon S3 equivalent). 

What I have to do is to restart the maxwell and insert information to the bootstrap table to re-sync all the missing data. But the bootstrap solution can only complement the updated data or new insert data, it cannot complement the deletions during the maxwell broken period.
